dev/tests/api-functional/testsuite/Magento/GraphQl/Store/StoreConfigResolverTest.php

```diff
@@ -70,27 +76,85 @@ public function testGetStoreConfig()
 QUERY;
         $response = $this->graphQlQuery($query);
         $this->assertArrayHasKey('storeConfig', $response);
-        $this->assertEquals($storeConfig->getId(), $response['storeConfig']['id']);
-        $this->assertEquals($storeConfig->getCode(), $response['storeConfig']['code']);
-        $this->assertEquals($storeConfig->getLocale(), $response['storeConfig']['locale']);
-        $this->assertEquals($storeConfig->getBaseCurrencyCode(), $response['storeConfig']['base_currency_code']);
+        $this->validateStoreConfig($defaultStoreConfig, $response['storeConfig']);
+    }
+
+    /**
+     * @magentoApiDataFixture Magento/Store/_files/store.php
+     * @magentoConfigFixture default_store store/information/name Default Store
+     * @magentoConfigFixture test_store store/information/name Test Store
+     * @throws Exception
+     */
+    public function testAvailableStoreConfigs(): void
```

Comment:
I think this should be in a separate test file since it's testing a different query/resolver